tests/Unit/Domain/Test_Orientations.cpp

```diff
@@ -0,0 +1,181 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
```

Comment:
Note I originally said "copy _and_ move". I don't think the meaning of equality of smart pointers is at all unclear. Options yes, though I don't think they should have a copy operation either then.